### PR TITLE
feat(producerjson): add original producerjson smart contract

### PR DESCRIPTION
**Pull request by xavier506**

### CODE REVIEW

File: producerjson/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# producerjson
+
+Producer JSON smart contract for EOS.
+
+# Usage
+
+### View the table
+
+```
+cleos get table producerjson producerjson producerjson
```
Comment (leisterfrancisco) [inline producerjson/README.md:10]: cleos `-u <endpoint>` get table producerjson producerjson producerjson

File: producerjson/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+```
+
+### Add to the table
+
+```
+cleos push action producerjson set '{"owner":"your_account", "json": "your_json"}' -p your_account@active
```
Comment (leisterfrancisco) [inline producerjson/README.md:16]: cleos `-u <endpoint>` ...

File: producerjson/README.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+```
+
+**Example**:
+```
+cleos push action producerjson set '{"owner":"teamgreymass", "json": "'`printf %q $(cat bp.json | tr -d "\r")`'"}' -p teamgreymass@active
```
Comment (leisterfrancisco) [inline producerjson/README.md:21]: cleos `-u <endpoint>` ...

File: producerjson/README.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+```
+
+### Remove from the table
+
+```
+cleos push action producerjson del '{"owner":"your_account"}' -p your_account@active
```
Comment (leisterfrancisco) [inline producerjson/README.md:27]: cleos `-u <endpoint>` ...

File: producerjson/README.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+---
+
+# How to build
+```
+./build.sh
+```
+or
+```
+eosiocpp -g producerjson.abi producerjson.cpp && eosiocpp -o producerjson.wast producerjson.cpp
```
Comment (leisterfrancisco) [inline producerjson/README.md:39]: What compile are we using here?

https://developers.eos.io/welcome/latest/getting-started-guide/hello-world/#procedure-to-compile-and-deploy

File: producerjson/README.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+```
+
+# How to deploy
+
+```
+cleos set contract producerjson producerjson -p producerjson@active
```
Comment (leisterfrancisco) [inline producerjson/README.md:45]: cleos `-u <endpoint>` ...

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

**Review by xavier506 — COMMENTED** [review]
LGTM

**Pull request merged**